clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/converters/DeployAppengineAtomicOperationConverter.groovy

```diff
@@ -22,16 +22,43 @@ import com.netflix.spinnaker.clouddriver.appengine.deploy.ops.DeployAppengineAto
 import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation
 import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperations
 import com.netflix.spinnaker.clouddriver.security.AbstractAtomicOperationsCredentialsSupport
+import com.netflix.spinnaker.clouddriver.appengine.deploy.exception.AppengineDescriptionConversionException
 import org.springframework.stereotype.Component
+import org.springframework.beans.factory.annotation.Autowired
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
 
 @AppengineOperation(AtomicOperations.CREATE_SERVER_GROUP)
 @Component
 class DeployAppengineAtomicOperationConverter extends AbstractAtomicOperationsCredentialsSupport {
+  @Autowired
+  ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
No need to initialize this ObjectMapper, we do that on startup and then add it to the Spring object registry.